examples/enumerations/enumerations.bal

```diff
@@ -10,7 +10,9 @@ import ballerina/io;
 //
 // `type Color RED|GREEN|BLUE;`
 enum Color {
-    RED, GREEN, BLUE
+    RED,
+    GREEN,
+    BLUE
 }
 
```

Comment:
Ballerina formatter does not allow writing enums with members in-line. I think we need to allow it. Created ballerina-platform/ballerina-lang#34896. Anyway, changed the bal according to the current formatting.